settings/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="surah_calligraphy">Surah Calligraphy</string>
+    <string name="aya_calligraphy">Aya Calligraphy</string>
```

Comment:
I think `Calligraphy` is a technical term

Reply:
Let's do it when the designs arrive